bofire/data_models/strategies/predictives/active_learning.py

```diff
@@ -0,0 +1,46 @@
+from typing import Dict, List, Literal, Optional, Type
+
+from pydantic import Field, field_validator
+
+from bofire.data_models.acquisition_functions.api import (
+    AnyActiveLearningAcquisitionFunction,
+    qNegIntPosVar,
+)
+from bofire.data_models.domain.api import Domain
+from bofire.data_models.features.api import CategoricalOutput, Feature
+from bofire.data_models.objectives.api import Objective
+from bofire.data_models.strategies.predictives.botorch import BotorchStrategy
+
+
+class ActiveLearningStrategy(BotorchStrategy):
+    type: Literal["ActiveLearningStrategy"] = "ActiveLearningStrategy"
+    acquisition_function: AnyActiveLearningAcquisitionFunction = Field(
+        default_factory=lambda: qNegIntPosVar()
+    )
+    weights: Optional[Dict[str, float]] = Field(default_factory=lambda: None)
```

Comment:
```suggestion
    weights: Optional[Dict[str, PositiveFloat]] = Field(default_factory=lambda: None)
```

Concerning the weights:
- you can import `PositiveFloat` from pydantic
- I think we concluded that we want to have the `weights` attribute under the acquisition function `qNegIntPostVar` as attribute as it is specific to this acquisition function and other active learning acquisition functions would not use this. So can you move it?
- We need then a `model_validator` with mode `"after"` in which we check that if `weights` are provided, the keys match the keys of the output features:
```python
@model_validator(mode="after")
def validate_acquisition_function(self):
   if isinstance(self.acquisition_function, qNegIntPosVar):
       if self.acquisition_function.weights is not None:
           if sorted(self.acquisition_function.weights.keys()) != sorted(self.domain.outputs.get_keys()):
               raise ValueError(balabala)
   return self
```